rector.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Rector\Core\Configuration\Option;
+use Rector\Core\ValueObject\PhpVersion;
+use Rector\PHPUnit\Set\PHPUnitSetList;
+use Rector\Set\ValueObject\SetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+    $services = $containerConfigurator->services();
+
+    $parameters->set(Option::PATHS, [__DIR__ . '/src']);
+
+    $parameters->set(Option::BOOTSTRAP_FILES, [
+        __DIR__ . '/vendor/autoload.php',
+    ]);
+
+    // basic rules
+    $parameters->set(Option::AUTO_IMPORT_NAMES, true);
+    $parameters->set(Option::IMPORT_DOC_BLOCKS, true);
+    $parameters->set(Option::IMPORT_SHORT_CLASSES, false);
+
+
+    $containerConfigurator->import(SetList::PHP_54);
+    $containerConfigurator->import(SetList::PHP_55);
+    $containerConfigurator->import(SetList::PHP_56);
+    $containerConfigurator->import(SetList::PHP_70);
+    $containerConfigurator->import(SetList::PHP_71);
+    $containerConfigurator->import(SetList::PHP_72);
```

Comment:
One lazy tip: https://twitter.com/rectorphp/status/1446103241817837573

Reply:
Thank you!